com.jetbrains.IntelliJ-IDEA-Ultimate.appdata.xml

```diff
@@ -2,7 +2,7 @@
 <component type="desktop-application">
   <id>com.jetbrains.IntelliJ-IDEA-Ultimate.desktop</id>
   <metadata_license>CC-BY-SA</metadata_license>
-  <project_license>Proprietary</project_license>
```

Comment:
Both of these is wrong its: `LicenseRef-proprietary`

Reply:
Actually, I wasn't sure about `project_license` and `metadata_license`. I was digging `https://spdx.org/licenses/` for something similar...but couldn't find anything.

I put `Commercial` in the end. Are you suggesting to change the `project_license` tag to `LicenseRef-proprietary`? What about the `metadata_license` tag?

Reply:
metadata should be `CC0-1.0` assuming you have rights to it